keyboards/mmd/km75/keyboard.json

```diff
@@ -0,0 +1,250 @@
+{
+    "manufacturer": "MMD",
+    "keyboard_name": "MMD-KM75",
+    "maintainer": "Smartmmd",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B3", "pin_b": "A7", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["B0", "C15", "C14", "C13", "A15", "B7", "B6", "B5", "B4", "B12", "B13", "B14", "A14", "B2", "B15"],
+        "rows": ["A0", "A1", "A2", "A3", "A4", "A5"]
+    },
+    "processor": "STM32F103",
+    "rgb_matrix": {
+        "animations": {
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_sat": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "band_val": true,
+            "breathing": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "cycle_up_down": true,
+            "digital_rain": true,
+            "dual_beacon": true,
+            "gradient_left_right": true,
+            "gradient_up_down": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "multisplash": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "solid_multisplash": true,
+            "solid_reactive": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_multinexus": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_simple": true,
+            "solid_reactive_wide": true,
+            "solid_splash": true,
+            "splash": true,
+            "typing_heatmap": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [5, 5], "x": 93, "y": 6, "flags": 4},
```

Comment:
RGB Matrix layout is incorrect.
![image](https://github.com/user-attachments/assets/c0c3919e-1b88-47d1-b45b-6b163d71351b)

Reply:
updated :）

Reply:
Looks better, although the placement still seems not quite right:
![image](https://github.com/user-attachments/assets/d53b896e-e811-4e4f-bc1f-e836fac63302)
And I am not sure what's going on in the top right. Is the encoder button the key at 0,14?

Reply:
Yes , there is button @ 0 ,14
BTW, how do youget the layout view? I can use to check before push/pull. Thanks